README.md

```diff
@@ -2,15 +2,59 @@
 
 This package was originally developed to help release [Embroider](https://github.com/embroider-build/embroider) and was extracted so everyone can use it 🎉
 
-## Installation
+## Features
 
+- no manual steps[^after-initial-setup]
+  - release from _anywhere_[^release-anywhere]
+- everything is automated via GH Action Workflows[^gh-by-default]
+- no need to give maintainers access to npm[^npm-config]
+- pnpm and monorepo support _by default_
+  - also supports non-pnpm repos
+- release preview PR 
+  - release via merge of the releae preview PR
+- automatic changelog
+  - changelog entries editable by editing PR titles
+
+This allows folks with GH Maintainer access to preview and release to npm without actually needing to have any keys locally on their machine (or access to the npm package, which makes management of the repo separate from the npm package easy and encouraged).
+
+[^gh-by-default]: While GitHub Actions is the default setup via [`create-release-plan-setup`][gh-create]
+[^after-initial-setup]: after initial setup, which is already mostly automated.
+[^npm-config]: The NPM token can be granular, and the package needs to have its "Publishing access" configured to either "Don't require two-factor authentication" or, "Require two-factor authentication or an automation or granular access token" (recommended). This can be configured under `https://www.npmjs.com/package/{package-name}/access`
+[^release-anywhere]: You can release from bed, the mall, the percelain, anywhere! No need for a computer! (assuming you have a pocket computer (phone))
```

Comment:
```suggestion
[^release-anywhere]: You can release from bed, the mall, the porcelain, anywhere! No need for a computer! (assuming you have a smart phone)
```

Reply:
what's wrong with pocket computer? :sweat_smile:

Reply:
lol I don't like brackets inside brackes 😄

Reply:
I think we should all start calling them pocket computers, as none of ever actually use the phone part :sweat_smile: